lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -389,6 +389,7 @@
 
 
 def create_block_device_meta(module, volume):
+    # TODO seems unused, remove?
```

Comment:
Yeah, +1 to remove that. It looks like it was added 4 years ago and wasn't even used then, and handling the IOPS ratio would be a new feature to the module (they were referring to `ratio has been increased from 10:1 to 30:1` on https://aws.amazon.com/about-aws/whats-new/2013/10/09/ebs-provisioned-iops-maximum-iops-gb-ratio-increased-to-30-1/). IOPS is specified as a value in volume and relevant values for the module are ranges from 100-20000 IOPS for io1 volumes and 100-10000 IOPS for gp2 volumes.